Clockcult is penalized for reciting scripture offstation

ChangelingRain opened the pull request:
:cl: Joan
rscadd: Reciting scripture while not on the station, centcom, or mining/lavaland will double recital time and component costs.
/:cl:

"But this isn't that much", you say. Spending twice and many components and twice as much time for anything offstation is massive, especially given conversion is affected by this.

ChangelingRain committed:
Clockcult is penalized for using scripture offstation

ChangelingRain committed:
no message

PKPenguin321 commented:
Do solars count as on or off station

ChangelingRain commented:
On station but not particularly practical
most humans need air and clockwork armor isn't very good against pressure

RemieRichards commented:
I wouldn't grant lavaland that freedom, it's already a hotspot cult base, don't make it more mechanically viable.

ChangelingRain committed:
rrrrr

ChangelingRain committed:
check, singular, because there's one

Tacolizard commented:
this kills tactical spacebases. seriously spacebases were a really cool aspect of cuck cult, and why would you penalize this for any other reason than nerfing that? is there some other reason for this?

ExcessiveUseOfCobblestone commented:
you can still make space bases, they just have to be on the station Zlevel

ChangelingRain committed:
WASTEFUL
DON'T WASTE MY COGS ON THIS
HOW DOES THE JUSTICAR GET ANYTHING DONE WITH SERVANTS LIKE YOU
YOU DISGRACE OUR MASTER'S NAME WITH THIS ENDEAVOUR
BOSS SAYS STOP WASTING TIME

Anonmare commented:
Eh, I'm fine with it. Don't particularly care either way tbh

ExcessiveUseOfCobblestone commented on code/game/gamemodes/clock_cult/clock_scripture.dm:
```diff
@@ -35,8 +35,19 @@ Judgement: 12 servants, 5 caches, 300 CV, and any existing AIs are converted or
+	"The barrier isn't weakened enough to make this practical.", "Don't waste alloy.")
+	var/static/list/nzcrentr_penalty = list("You'd be easy to hunt in that little hunk of metal.", "Boss says you need to get back to the beacon.", "Boss says I can kill you if you do this again.", \
+	"Sending you power is too difficult here.", "Boss says stop wasting time.")
+
```
❤️ it

ChangelingRain committed:
%

Iamgoofball commented on code/game/gamemodes/clock_cult/clock_scripture.dm:
```diff
@@ -35,8 +35,19 @@ Judgement: 12 servants, 5 caches, 300 CV, and any existing AIs are converted or
+	var/static/list/nezbere_penalty = list("You disgrace our master's name with this endeavour.", "This is too far from the station to be a good base.", "This will take too long, friend.", \
+	"The barrier isn't weakened enough to make this practical.", "Don't waste alloy.")
+	var/static/list/nzcrentr_penalty = list("You'd be easy to hunt in that little hunk of metal.", "Boss says you need to get back to the beacon.", "Boss says I can kill you if you do this again.", \
+	"Sending you power is too difficult here.", "Boss says stop wasting time.")
```
He's coming too?
